### PR TITLE
updated Wordpress example to be easier to follow, added/updated images

### PR DESCRIPTION
**Pull request by londoncalling**
Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by londoncalling** [comment]
@moxiegirl @dnephin @shin- please have a look at these updates to the WordPress example docs. Daniel and I found the previous docs a little difficult to follow, so we changed a couple of things in the manually-created files. I've tested this, and it works great. I also added some screen snaps, and clarified what you get and where you are at the end of the example. If you have 5 mins, please try out setting up and building the sample app to test the new docs. Let me know if it's good to go. 

Thanks! Vicky

**Comment by dnephin** [comment]
LGTM

File: docs/wordpress.md [hunk 1]
```diff
@@ -13,85 +13,126 @@ weight=6
...
+
+    This project directory will contain a `Dockerfile`, a `docker-compose.yaml` file, along with a downloaded `wordpress` directory and a custom `wp-config.php`, all of which you will create in the following steps.
+
+2. Change directories into your project directory.
+
+    For example, if you named your directory `my_wordpress`:
+
+        $ cd my-wordpress/
+
+3. Create a `Dockerfile`, a file that defines the environment your app is going to run in. For more information on how to write Dockerfiles, see the [Docker user guide](https://docs.docker.com/engine/userguide/dockerimages/#building-an-image-from-a-dockerfile) and the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/). In this case, your Dockerfile should include these two lines:
```
Comment (moxiegirl) [inline docs/wordpress.md]: nit: Para break after run in.

Comment (moxiegirl) [inline docs/wordpress.md]: Docker Engine user guide  -> trying to specific and not generalize everything under a Docker label.

"Docker container" is a Docker resources.

File: docs/wordpress.md [hunk 2]
```diff
@@ -13,85 +13,126 @@ weight=6
...
+
+        FROM orchardup/php5
+        ADD . /code
+
+    This tells Docker how to build an image defining a container that contains PHP and WordPress.
```
Comment (moxiegirl) [inline docs/wordpress.md]: This tells the Docker Engine daemon or Engine daemon.

File: docs/wordpress.md [hunk 3]
```diff
@@ -13,85 +13,126 @@ weight=6
 # Quickstart: Compose and WordPress
 
 You can use Compose to easily run WordPress in an isolated environment built
```
Comment (moxiegirl) [inline docs/wordpress.md]: Docker Compose first use on the page and Compose after.

**Comment by moxiegirl** [comment]
Minor comments mostly around disambiguation around products. See

https://docker.atlassian.net/wiki/display/DOCSTEAM/Terminology

**Comment by londoncalling** [comment]
@dnephin @shin- , @moxiegirl 's comments are in, and you all can verify on GitHub or at http://104.131.43.236:8001/ 

Can someone merge this PR when the build completes? I think we're good :+1:  :-)

**Comment by londoncalling** [comment]
@dnephin would you mind merging this PR? I think someone else besides me should do it. All good.

**Pull request merged**